adoc/chapters/programming_interface.adoc

```diff
@@ -52,6 +52,8 @@ _backend_name_ is the associated identifier from [code]#backend# in
 all upper-case.  See <<chapter.extensions>> for the name of the macro
 if the vendor defines the <<backend>> outside of the Khronos SYCL group.
 
+If a backend listed in [code]#enum class backend# is not available at compile
+time, the associated macro will be left undefined.
```

Comment:
This seems to contradict lines 33—35:

> The [code]#enum class backend# is implementation-defined and must be
> populated with a unique identifier for each <<backend>> that the SYCL
> implementation supports.

If the back-end is not available, it is not supported and cannot be in the `enum`.

Reply:
Good point, I've attempted to update the other sections to reflect the changes as well.